recipe/meta.yaml

```
@@ -67,12 +67,12 @@ outputs:
         - pip
         - python
       run:
-        - python
-        - packaging
+        - python >=3.8
```

```suggestion
        - python
```
We don't have a build for python < 3.8. See `build` section.